src/device_trezor/trezor/update_protob.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+
+cd "$(dirname "$0")"
+
+if [ ! -d "trezor-common" ]; then
+  git clone https://github.com/trezor/trezor-common.git
+fi
+
+cd trezor-common
+git fetch
+git reset --hard bc28c316d05bf1e9ebfe3d7df1ab25831d98d168
```

Comment:
The script itself seems fine, but I find its name confusing: This fetches a particular version of the protocol files (or resets them), whereas update usually implies retrieving a newer version.

Reply:
I changed the name of the script to `fetch_protob.sh`.